clients/besu/besu.sh

```diff
@@ -124,25 +124,31 @@ else
     FLAGS="$FLAGS --network-id=1337"
 fi
 
-# Sync mode.
-case "$HIVE_NODETYPE" in
-    "" | "full")
-        FLAGS="$FLAGS --sync-mode=FAST --fast-sync-min-peers=1 --Xsynchronizer-fast-sync-pivot-distance=0" ;;
-    "light")
-        echo "Ignoring HIVE_NODETYPE == light: besu does not support light client" ;;
-esac
+# Sync mode, must only be changed for non merge related tests.
+if [ "$HIVE_TERMINAL_TOTAL_DIFFICULTY" == "" ]; then
+    if [ "$HIVE_NODETYPE" == "light" ]; then
+        echo "Ignoring HIVE_NODETYPE == light: besu does not support light client"
+    else
+        FLAGS="$FLAGS --sync-mode=FAST --fast-sync-min-peers=1 --Xsynchronizer-fast-sync-pivot-distance=0"
+    fi
+fi
 
 # Configure RPC.
-RPCFLAGS="--host-whitelist=*"
+RPCFLAGS="--host-allowlist=*"
 if [ "$HIVE_GRAPHQL_ENABLED" == "" ]; then
     RPCFLAGS="$RPCFLAGS --rpc-http-enabled --rpc-http-api=ETH,NET,WEB3,ADMIN --rpc-http-host=0.0.0.0"
 else
-    RPCFLAGS="$RPCFLAGS --rpc-http-port=8550" # work around duplicate port error
     RPCFLAGS="$RPCFLAGS --graphql-http-enabled --graphql-http-host=0.0.0.0 --graphql-http-port=8545"
 fi
 
 # Enable WebSocket.
-RPCFLAGS="$RPCFLAGS --rpc-ws-enabled --rpc-ws-api=ETH,NET,WEB3,ADMIN --rpc-ws-host=0.0.0.0"
+#RPCFLAGS="$RPCFLAGS --rpc-ws-enabled --rpc-ws-api=ETH,NET,WEB3,ADMIN --rpc-ws-host=0.0.0.0"
```

Comment:
No WebSocket anymore?

Reply:
Websocket is still in place for the "main" stack. Only the engine api is on a combined http stack that upgrades from http to websockets.

Reply:
Ahh, so `--rpc-ws-enabled` is not needed anymore to enable it? I was just wondering why you commented out this flag here.

Reply:
I have tested this, and WebSocket is no longer enabled on besu with this change.